src/game/client.cpp

```diff
@@ -291,6 +291,99 @@ namespace client
     }
     ICOMMAND(0, writevars, "sii", (char *name, int *all, int *sv), if(!(identflags&IDF_WORLD)) writegamevars(name, *all!=0, *sv!=0));
 
+    void write_weapon_vars(const char* weapon_name, const char* file_name, bool all = false, bool server = false)
```

Comment:
Document what is going on in this function and what the parameters mean.

Reply:
This function seems new; how about using `std::string` directly? It'll make things easier most likely.